thunder/executors/nvfuserex_impl.py

```diff
@@ -1615,6 +1615,15 @@ def real(a: TensorProxy | Number, *, fd: FusionDefinition, lc_to_nv_map: dict) -
 register_supported(PrimIDs.REAL, real, _elementwise_unary_check)
 
 
+def imag(a: TensorProxy | Number, *, fd: FusionDefinition, lc_to_nv_map: dict) -> Any:
+    nva = getnv(a, fd, lc_to_nv_map)
+
+    return fd.ops.real(nva)
```

Comment:
This looks like a typo. How is nvFuser passing the tests if it's taking the real part and not the imaginary part from the tensor? Is it not being tested properly?

Reply:
It's not passing exactly, operator consistency tests with complex input are skipped: https://github.com/Lightning-AI/lightning-thunder/blob/858a54d331159a266c586b626a3c78bc382f3ea8/thunder/tests/test_ops.py#L56

Fully enabling them would increase the CI run time substantially.  Should there be some sort of flag in OpInfo to opt in to testing with complex dtypes?

Reply:
That's right! Thank you for reminding me.

@beverlylytle, Would you file an issue to allow passing an option to run the complex tests, and a separate issue to allow a test to specify that its complex variants should not be skipped?